examples/arkode/CXX_serial/CMakeLists.txt

```diff
@@ -28,12 +28,13 @@ set(ARKODE_examples
     "ark_kpr_Mt.cpp\;0 4 1\;develop"
     "ark_kpr_Mt.cpp\;0 -4\;exclude-single"
     "ark_kpr_Mt.cpp\;1 -5\;exclude-single"
-    "ark_kpr_Mt.cpp\;2 -5 0 -10\;develop"
+    "ark_kpr_Mt.cpp\;2 -5 0 -10\;exclude-single"
```

Comment:
I dont see any changes in the `ark_kpr_Mt.cpp` example. What is the reason for this change?

Reply:
This was likely fixed with the update elsewhere.  I'll add it back in to see if it now works.

Reply:
It turns out that this was an explicit fixed-step run and the [irrelevant] tolerances were set too tightly for single-precision, causing ARKODE to return with an error that "too much accuracy was requested".  I'm not sure why other branches do not get tripped up by the same issue, but it's clear that in this configuration the test for "too much accuracy" should actually be disabled.  I've done that in `src/arkode/arkode.c`, and the single-precision test should now pass.